docs/development/Building in Windows 10 or 11 with Linux Subsystem.md

```diff
@@ -36,7 +36,7 @@ To run  `cmake` in the latest version you will need to update from Ubuntu `18_04
 
 Mount MS windows C drive and clone INAV
 1.   `cd /mnt/c`
-1.   `git clone https://github.com/iNavFlight/inav.git`
+1.   `git clone https://github.com/iNavFlight/inav.git` (latest code) / `git clone -b 6.1.1 --single-branch https://github.com/iNavFlight/inav.git` (option "-b" for downloading specific releases - version 6.1.1 in this example)
```

Comment:
This is slightly misleading. Because `git clone https://github.com/iNavFlight/inav.git` should always the first step, but then

`git checkout 6.1.1` to switch to 6.1.1 tag. and then `git checkout -b my-branch` to make own branch from it
`git

Reply:
Agree - proposed new text in patch-1